v3/docs/TUTORIAL.md

```diff
@@ -1593,16 +1593,16 @@ async def bar(lock):
 ```
 As with normal context managers an exit method is guaranteed to be called when
 the context manager terminates, whether normally or via an exception. To
-achieve this the special methods `__aenter__` and `__aexit__` must be
+achieve this, the special methods `__aenter__` and `__aexit__` must be
 defined, both being tasks waiting on a task or `awaitable` object. This example
 comes from the `Lock` class:
 ```python
-    async def __aenter__(self):
-        await self.acquire()  # a coro defined with async def
```

Comment:
The indentation here was deliberate: as a bound method it would normally be indented.